docs/ref/contrib/admin/index.txt

```diff
@@ -2031,6 +2033,10 @@ templates used by the :class:`ModelAdmin` views:
 Other methods
 ~~~~~~~~~~~~~
 
+.. method:: ModelAdmin.autocomplete_view(request)
+
+    Django view for the page that shows the autocomplete results. See note below
```

Comment:
What note below?

Reply:
Right after defining all the methods in the `Other methods`  section, there is a small note explaining that these views `render the pages that deal with model instances CRUD operations. As a result, completely overriding these methods will significantly change the behavior of the admin application.`. The same comment is added for almost all of the other methods in the section. 

I agree it is debatable whether the below note applies to this view, as this is not a view that renders pages that handles CRUD operations. However, I consider at least the other half of the note to be applicable to this view as well (overriding this will change the behaviour of the admin page).

@timgraham, what's your take on this?

Reply:
If I had to guess, Tim may have been looking at a GitHub diff and neglected to notice the identical "see note below" below. But let's standardize it with a trailing period, like the other five.

Reply:
> I agree it is debatable whether the below note applies to this view, as this is not a view that renders pages that handles CRUD operations.

Providing autocomplete hints is part of a create/update workflow, so I think we're fine here. Shouldn't cause confusion IMO.